.clang-format

```diff
@@ -3,3 +3,11 @@ ColumnLimit: 90
 DerivePointerAlignment: false
 IndentCaseLabels: false
 PointerAlignment: Right
+BinPackArguments: false
+BinPackParameters: false
+AllowAllArgumentsOnNextLine: false
+AllowAllParametersOfDeclarationOnNextLine: false
+InsertTrailingCommas: Wrapped
```

Comment:
This option seems only useful for Javascript? https://clang.llvm.org/docs/ClangFormatStyleOptions.html

Reply:
Thanks. I'll revisit all configurations.